Clarify how an APM host is defined (#11610)
DataDog/documentation: content/en/account_management/billing/pricing.md

## APM

> * If a host (as defined by [Infrastructure monitoring](#infrastructure-monitoring)) receives traces from an instrumented application, it's counted as one **APM host**.

Reviewer: ```suggestion
* If a host (defined in [Infrastructure monitoring](#infrastructure-monitoring)) receives traces from an instrumented application, Datadog counts one **APM host**.
```

  * On a high watermark plan (HWMP), the hourly measurements are ordered from highest to lowest at the end of the month, and Datadog charges based on the eighth highest measurement.

>   * On a hybrid monthly/hourly plan (MHP), Datadog charges your minimum monthly commitment, and for any host hours above that commitment, Datadog charges an hourly rate.

Reviewer: ```suggestion
  * On a hybrid monthly and hourly plan (MHP), Datadog charges your minimum monthly commitment, and for any host hours above that commitment, Datadog charges an hourly rate.
```
